src/test/java/com/google/cloud/discotoproto3converter/DiscoToProto3ConverterAppTest.java

```diff
@@ -46,6 +46,26 @@ public void convert() throws IOException {
     assertEquals(baselineBody, actualBody);
   }
 
+  @Test
+  public void convertWithIgnorelist() throws IOException {
+    DiscoToProto3ConverterApp app = new DiscoToProto3ConverterApp();
+    Path outputDir = Files.createTempDirectory("disco-to-proto3-converter");
```

Comment:
Since we are starting to have multiple tests here, please consider moving the setup logic (like creating the temp dir) to the `public void setUp()` method with `@org.junit.Before` annotation, and then tearDown() method with `@After` annotation (which removes the temp dir).

Reply:
I added `deleteOnExit()`, which can be done in `setUp`, so I did not add a `tearDown` method. If this isn't right, I can change it!